engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1315,6 +1312,26 @@ public void orchestrateStart(final String vmUuid, final Map<VirtualMachineProfil
         }
     }
 
+    private void logUefiParameters(Map<VirtualMachineProfile.Param, Object> params) {
```

Comment:
@DaanHoogland is it possible params = null ?

Reply:
/me checking

Reply:
good spot, tnx